src/main/scala/me/rexim/morganey/Commands.scala

```diff
@@ -61,6 +60,8 @@ object Commands {
         val message = removed match {
           case Nil       => "No bindings were removed!"
           case hd :: Nil => s"Binding '$hd' was removed!"
+          case bindings if bindings.size > unbindCountThreshold =>
+            s"${bindings.size} bindings were removed!"
           case in :+ ls  =>
             val bindingEnumeration = in.map(b => s"'$b'").mkString(", ") + s" and '$ls'"
```

Comment:
A nitpick: there should be a trailing comma here (so-called [_Oxford comma_](https://en.wikipedia.org/wiki/Serial_comma)) before `and`.